features/people.feature

```diff
@@ -14,7 +14,7 @@ Scenario: Viewing all people
   And I should see that "Fred Bloggs" is listed under "b"
 
 Scenario: Viewing the person page for a minister
-  Given "Benjamin Disraeli" is a minister with a history
+  Given a person called "Benjamin Disraeli"
```

Comment:
The person this step creates will not have any roles, which means the scenario title is no inaccurate (they're no longer a minister).